substra/cli/interface.py

```diff
@@ -1038,6 +1038,27 @@ def run_local(algo, train_opener, test_opener, metrics, rank,
                    inmodels=inmodels)
 
 
+@cli.group()
+@click.pass_context
+def cancel(ctx):
+    """Cancel execution of an asset."""
+    pass
+
+
+@cancel.command('compute_plan')
+@click.argument('compute_plan_id', type=click.STRING)
+@click_global_conf
+@click.pass_context
+def cancel_compute_plan(ctx, compute_plan_id):
+    """Cancel execution of a compute plan."""
+    client = get_client(ctx.obj)
+    # method must exist in sdk
+    method = getattr(client, f'cancel_compute_plan')
+    res = method(compute_plan_id)
```

Comment:
Why are you using a `getattr` method here? It seems not needed.

Reply:
You're absolutely right, I'll remove it